.semaphore/semaphore.yml

```diff
@@ -56,34 +34,22 @@ blocks:
             - sem-version java 8 # NonDex only supports Java 8
             - .ci/validation.sh nondex
 
-        - name: No error test on pmd
-          priority:
-            - value: 80
-              when: true
-          commands:
-            - mvn -e install -Pno-validations
-            - .ci/validation.sh no-error-pmd
-
-        - name: No error test on Configurate
-          priority:
-            - value: 80
-              when: true
-          commands:
-            - sem-version java 14
-            - mvn -e install -Pno-validations
-            - .ci/validation.sh no-violation-test-configurate
-
-        - name: No violation testing on josm
-          commands:
-            - mvn -e install -Pno-validations
-            - .ci/validation.sh no-violation-test-josm
-
-        - name: No exception test
+        - name: Validation (openjdk11)
           matrix:
-            - env_var: PROJECT
+            - env_var: CMD
               values:
-                - guava-with-google-checks
-                - guava-with-sun-checks
-          commands:
-            - mvn -e install -Pno-validations
-            - .ci/no-exception-test.sh $PROJECT
+                - .ci/no-exception-test.sh guava-with-google-checks
+                - .ci/no-exception-test.sh guava-with-sun-checks
+                - .ci/validation.sh all-sevntu-checks
+                - .ci/validation.sh check-missing-pitests
+                - .ci/validation.sh eclipse-static-analysis
+                - .ci/validation.sh no-error-pmd
```

Comment:
`check-missing-pitests` was assigned priority 10 originally, and `no-error-pmd` was assigned 80. Now, priority assignment will be automatic from semaphore. Out of all these tests, `no-error-pmd` is the most frequently failed one for me, so I think that it made sense for it to have higher priority so that the build fails faster and not waste time/resources. I am not sure how many builds we can run in parallel, and if this even matters, but I thought it would be good to mention.

Reply:
Makes sense. I'll make two jobs: one with high priority, another with default priority